### ISSUE TITLE
Reverse Engeneering

### ISSUE THREAD

**Issue by dschense** [tobadia/petGPS#14]
Hey, nice Project! These little Trackers are awesome in size and function. The only thing I hate is the hardcoded stuff like the default server for fallback. Anybody who ever tried reverse engeneering these little pcbs? extracting firmware, flashing firmware, modify the firmware, remove the hard coded Server IP, add or modify commands? 
Ive searched the whole internet, but seems like nobody ever did this step.

**Comment by samex** [comment]
Hey, are you have checked with a sniffer which kind of out/incoming connections appear?

Its not possible to change 2 servers with the communication protocoll via sms?

**Comment by samex** [comment]
@tobadia thanks for the reply. I asked, is there any hardcoded fallback server that u cant change by sms command? How do I know there are 1 or 2 servers which you can change by sms, how I understand @dschense says there is one more server beyond the standard servers, is that true?

**Comment by tobadia** [comment]
Yes, there is one. It's the url for... 365gps.com, or something like that ? Basically it reverts to that when your custom server doesn't respond after a while. 

Good catch for SERVER vs DOMAIN keywords. I honestly have no idea if that works... To try it I'd first have to find again where I put my GPS. It's been a while... Likely in a drawer somewhere. Will look into it.

**Comment by Abhishek-op** [comment]
Hi mates

Thanks for your work
I have bought a 4g based tracker that is zx509. Port docs is same.
As I have got the info this is using ax905 v1.0.3 I think it is Bmobile ax905 firmware http://www.locationof.com/phones/bmobile-ax905/

I'm not sure but also from other forums I have seen it's like Android device without screen so we can log to chip using usb.
https://www.traccar.org/forums/topic/zx808-compatible/

**Comment by Abhishek-op** [comment]
And chip is arm based mediatech  Chip: MT2503+5931+3333

**Comment by dschense** [comment]
@Abhishek-op thanks for your comments. I tried to get on any of my trackers, but had no luck over usb connection using total control from sigma-rt.
How did you connect them? Did you need any driver to get the device connecting?

**Comment by Brandon-T** [comment]
https://github.com/FoxExe/ZX908/tree/main

This might help. This is for the ZX908 found via: https://www.traccar.org/forums/topic/zx9082/page/2/#post-109909
